src/wallet/init.cpp

```diff
@@ -46,7 +46,7 @@ void WalletInit::AddWalletOptions() const
 
     gArgs.AddArg("-fallbackfee=<amt>", strprintf("A fee rate (in %s/kB) that will be used when fee estimation has insufficient data. 0 to entirely disable the fallbackfee feature. (default: %s)",
                                                                CURRENCY_UNIT, FormatMoney(DEFAULT_FALLBACK_FEE)), ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
-    gArgs.AddArg("-keypool=<n>", strprintf("Set key pool size to <n> (default: %u)", DEFAULT_KEYPOOL_SIZE), ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
+    gArgs.AddArg("-keypool=<n>", strprintf("Set key pool size to <n> (default: %u). Smaller sizes may increase the risk of losing funds when restoring from an old backup, if none of the addresses in the original keypool have been used.", DEFAULT_KEYPOOL_SIZE), ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
```

Comment:
Now that I test it on the command line, perhaps:
```diff
- Smaller sizes may increase the risk of losing funds
+ Warning: A size lower than the default may increase the risk of losing funds
```

Reply:
> Now that I test it on the command line, perhaps:
> 
> ```diff
> - Smaller sizes may increase the risk...
> + Warning: A size lower than the default may increase the risk...
> ```

Adding the Warning: prefix could be a good idea, but it's misleading to say that "size lower than the default may increase the risk", because that implies the default value has the right amount of risk, even though it could easily be unsafe if you generate a lot of addresses. "Smaller sizes may increase the risk" is more accurate because smaller sizes decrease the risk and larger sizes increase it, and the default value is actually pretty arbitrary.

Reply:
Good points @ryanofsky; I agree.

Reply:
Thanks @jonatack for the suggestion took it on f41d589, should be only diff!

> "Smaller sizes may increase the risk" is more accurate because smaller sizes decrease the risk and larger sizes increase it

I think you mean the inverse here, at least that's what the updated comment aims to convey.